forms-flow-api/src/formsflow_api/services/theme.py

```diff
@@ -0,0 +1,40 @@
+"""This exposes theme service."""
+
+from formsflow_api_utils.utils.user_context import UserContext, user_context
+
+from formsflow_api.models import Themes
+from formsflow_api.schemas import ThemeCustomizationSchema
+
+
+class ThemeCustomizationService:
+    """This class manages theme service."""
+
+    @staticmethod
+    @user_context
+    def create_theme(data, **kwargs):
+        """Create new theme entry."""
+        theme_schema = ThemeCustomizationSchema()
+        user: UserContext = kwargs["user"]
+        data["created_by"] = user.user_name
+        data["tenant"] = user.tenant_key
+        theme_customization = Themes.create_theme(data)
+        result = theme_schema.dump(theme_customization)
+        return result
+
+    @staticmethod
+    def get_theme(tenant_key):
+        """Return theme using tenant key else default theme."""
+        theme = Themes.get_theme(tenant_key)
+        theme_schema = ThemeCustomizationSchema()
+        result = theme_schema.dump(theme)
```

Comment:
Initialize a common ThemeCustomizationSchema() object and utilize the object within functions